spec/components/govuk_component/configuration/phase_banner_component_configuration_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'spec_helper'
+
+RSpec.describe(GovukComponent::PhaseBannerComponent, type: :component) do
+  let(:phase) { 'Alpha' }
+  let(:text) { 'New service' }
+  let(:kwargs) { {} }
+
+  describe "configuration" do
+    after { Govuk::Components.reset! }
+
+    describe 'default phase banner component tag and text' do
+      let(:overridden_default_tag) { 'Beta' }
+      let(:overridden_default_text) { 'Public service' }
+
+      before do
+        Govuk::Components.configure do |config|
+          config.default_phase_banner_component_tag = overridden_default_tag
+          config.default_phase_banner_component_text = overridden_default_text
+        end
+      end
+
+      subject! { render_inline(GovukComponent::PhaseBannerComponent.new(**kwargs)) }
```

Comment:
I think it makes sense to drop `kwargs` here, and above (lines 4-6), they're not actually used for anything.